### DISCUSSION TITLE
Issue with Cloning and Running Hextra This Repository

### DISCUSSION THREAD

**Discussion by Welding-Torch** [imfing/hextra#33] [Q&A] [answered]
Hi, I'd like to clone this repo and quickly make a few edits for a different project of mine. I am aware of the Hextra Starter Template repo, but I wanna start with this repository instead because there's already content in place and it's easier to edit. However, when I clone this repo and run the following commands:
```
git clone https://github.com/imfing/hextra.git
cd hextra
hugo mod
# Start the server
hugo mod tidy
hugo server --logLevel debug --disableFastRender -p 1313
```
I see this output in the browser:
![image](https://github.com/imfing/hextra/assets/46340124/ad283b70-6e2d-4716-b94d-12e2a7a6c4b3)

Which is obviously an issue, so I was hoping for a way to fix this. Any ideas?

♥ Cheers, love this theme.

**Comment by imfing** [comment, marked as answer]
@Welding-Torch 
This repository is primarily designed for theme development. If you're interested in the site content, you'll find it in the `exampleSite` directory. The command required to run the development server is different in this case, as you can see in this specific line of our `package.json` file:

https://github.com/imfing/hextra/blob/8e8f7f23c955bd31f9faba501a2121d74e83793c/package.json#L4

I **strongly recommend against** running or modifying the content directly within this repository. For a more manageable approach, it's better to start with the template repository. You can then copy the content and configuration files from this `exampleSite` directory into your new project. This way, you maintain a clean separation between theme development and your own site content.

**Comment by Welding-Torch** [comment]
@imfing This command https://github.com/imfing/hextra/blob/8e8f7f23c955bd31f9faba501a2121d74e83793c/package.json#L4 worked perfectly, thank you!

Also from a quick go-through of this repository, I understand that the content of this site isn't store in `exampleSite` but it actually is stored in `layouts`. The homepage is [layouts/hextra-home.html](https://github.com/imfing/hextra/blob/main/layouts/hextra-home.html). 

If there's a way for me to copy all the content and dependencies for said content from this theme repository to the template repository, do let me know.

**Comment by imfing** [comment]
> If there's a way for me to copy all the content and dependencies for said content from this theme repository to the template repository, do let me know.

just need to copy the `exampleSite/content` and `exampleSite/hugo.yaml` to the root of the template repository

**Comment by Welding-Torch** [comment]
Hey, I followed your instructions and copied the above files to the root of the template repository, but I was unable to see the [layouts/hextra-home.html](https://github.com/imfing/hextra/blob/main/layouts/hextra-home.html) Homepage when I ran the server from the template. This is because the contents of https://imfing.github.io/hextra/ are not stored in the `exampleSite` folder but instead in the `layouts` folder. 

I think there may have been a misunderstanding, I would actually like the contents of the `layouts` folder as starter material for a different project. This is because the `layouts` folder includes a homepage, which the _hextra-starter-template_ does not have. 

Hope this clears up any confusion 👍.

**Comment by imfing** [comment]
Gotcha, you also need to copy the `layouts/hextra-home.html` to the `layouts` directory of your site, which will override the default hextra-home.
Make sure you set the layout property in the frontmatter of your _index.md file
